honeycomb-core/src/cmap/dim2/advanced_ops.rs

```diff
@@ -50,18 +50,31 @@ impl<T: CoordsFloat> CMap2<T> {
     ///  1             2    =>    1      3      2   | + denote darts that encode vertex IDs
     ///    <----2----+              <-4-- <-2-+     |
     /// ```
-    pub fn split_edge(&mut self, edge_id: EdgeIdentifier, midpoint_vertex: Option<T>) {
+    pub fn split_edge(
+        &mut self,
+        edge_id: EdgeIdentifier,
+        new_darts: (DartIdentifier, DartIdentifier), // 2D => statically known number of darts
+        midpoint_vertex: Option<T>,
+    ) {
```

Comment:
Update the documentation above with the new prototype.

It is not clear what `new_darts` is, and why the user has to pass these ids.

Reply:
I've added details about dart IDs in the documentation (c9b7ec9)